docs/importing-data.md

```diff
@@ -9,7 +9,7 @@ Data may be imported from Google Sheets into SQL tables. Columns in these spreed
 1. Create a new gsheet in the "Pegasus" Google Docs folder, naming your columns using type suffixes.
   - Make Site GoogleCloud (`site.googlecloud@code.org`) the owner of the gsheet.
 1. Add a new `cdo-table-name.gsheet` file locally in the `pegasus/data` folder.
-1. Paste the gsheet ID in a new corresponding `pegasus/data/cdo-table-name.gsheet` file. (See more context in [this PR](https://github.com/code-dot-org/code-dot-org/pull/54047))
+1. Paste the gsheet ID in the corresponding `pegasus/data/cdo-table-name.gsheet` file (from step 2). (See more context in [this PR](https://github.com/code-dot-org/code-dot-org/pull/54047))
```

Comment:
Should this PR be the new PR that's linked to be an example of pasting in the gsheet ID rather than creating a new corresponding file?

Reply:
Nope! I just changed the wording here since it's a continuation of the step above, and what was there originally made it sound like you have to make another new `.gsheet` file. That linked PR is the explanation for why it works this way.

Reply:
Oh gotcha!